### DISCUSSION TITLE
BUG - bad label of count lines

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#101] [Bugs and Reports]
when you wanna choose how much lines you wanna show of it still shows 50 results. 
![Uploading Snímek obrazovky 2024-11-28 v 23.03.06.png…]()

**Comment by bf-monster** [comment]
Hello, i can not load the image

**Comment by bf-gabagool** [comment]
Hey @JackHipeek , we can not load the image, if the issue still occurs, please can you try uploading another screenshot?

**Comment by bf-monster** [comment]
Closing the issue
